pages/_lang/reportCard.vue

```diff
@@ -262,154 +246,219 @@ main {
 #toolkitWrapper {
   animation-name: slideup;
   animation-duration: 300ms;
-}
-
-.cardBlock {
-  display: flex;
-  justify-content: space-between;
-}
 
-#toolkitSection {
-  margin-top: 36px;
-  display: flex;
-  align-items: center;
+  grid-column: 1 / span 12;
 
-  a {
-    margin-left: 10px;
-    color: black;
-    text-transform: uppercase;
-    font-size: 12px;
-    font-weight: bold;
-    border-bottom: solid 2px;
-  }
-
-  a:hover {
-    box-shadow: none;
-  }
 }
 
-#infoSection h2 {
-  margin-top: 36px;
+main {
+  @include grid;
+
+  margin-bottom: 2em;
 }
 
 h2 {
+  margin-top: 2em;
   font-size: 32px;
-  font-weight: bold;
+  font-weight: 700;
   margin-bottom: 17px;
 }
 
-p {
-  width: 395px;
-  margin-bottom: 30px;
+/* horizontal bar after heading */
+h2:after {
+  content: "";
+  display: block;
+
+  width: 19%; /* TODO: Not part of Grid */
+  padding-top: 17px;
+  border-bottom: solid 1px rgba(255, 255, 255, 0.3);
 }
 
 #inputSection {
-  width: 20em;
-  margin-top: 100px;
+  grid-column: 1 / span 5;
+
   color: white;
 
-  #backToOld {
-    font-size: 12px;
-    line-height: 18px;
-    margin-top: 20px;
-    margin-bottom: 0px;
-
-    a {
-      color: inherit;
-      box-shadow: none;
-      color: inherit;
-      text-decoration: underline;
+  display: grid;
+  grid-template-rows: 60% 40%;
+
+  #topHalfHome {
+    grid-row: 1;
+
+    form {
+      display: flex;
     }
-  }
 
-  #expertModeBlock {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
-    margin-top: 65px;
-    margin-right: 1em;
+    input {
+      background: transparent;
+      color: white;
 
-    #expertModeButton {
-      width: 200px;
+      padding-top: 13px;
+      padding-bottom: 12px;
       font-weight: bold;
       font-size: 18px;
       border: none;
+      border-bottom: solid 1px rgba(255, 255, 255, 0.4);
+      margin-right: 0.3em;
+      margin-top: 20px;
+      outline: none;
+      
+      &::placeholder {
+        color: white;
+      }
+
+      &:hover, &:focus {
+        border-bottom: solid 1px white;
+      }
+    }
+
+    #getStartedButton {
+      border: none;
+      font-weight: bold;
+      font-size: 18px;
       border-radius: 22px;
       padding-top: 9px;
       padding-bottom: 11px;
-      background-image: linear-gradient(to right, #7644c2, #11999e);
-      color: white;
+      padding-left: 23px;
```

Comment:
A lot of these changes are just re-organizing the CSS order to match the order of divs/elements in the HTML above to make it easier to find things. :)